index.html

```diff
@@ -3138,9 +3138,95 @@ <h2>DID Resolution</h2>
 consumption rules.">
         <figcaption>
 Functions resolve() and resolveRepresentation().
+See also: <a class="longdesc-link"
+href="#resolve-resolverepresentation-longdesc">narrative description</a>.
         </figcaption>
       </figure>
 
+        <div class="longdesc" id="resolve-resolverepresentation-longdesc">
+            <p>
+                The upper middle part of the diagram contains a rectangle with dashed grey outline, containing two
+                blue-outlined rectangles, one above the other, as follows.
+                The upper, larger rectangle is labeled, in blue, "Core Properties", and contains the following [INFRA]
+                notation:
+            </p>
+            <pre>
+«[
+  "id" → "example:123",
+  "verificationMethod" → « «[
+    "id": "did:example:123#keys-1",
+    "controller": "did:example:123",
+    "type": "Ed25519VerificationKey2018",
+    "publicKeyBase58": "H3C2AVvLMv6gmMNam3uVA"
+  ]» »,
+  "authentication" → «
+    "did:example:123#keys-1"
+  »
+]»
+            </pre>
+            <p>
+            The lower, smaller rectangle is labeled, in blue, "Core Representation-specific Entries (JSON-LD)", and
+                contains the following monospaced [INFRA] notation:
```

Comment:
`[INFRA]` should be changed to `[[INFRA]]` or `<a data-cite="INFRA#maps">INFRA</a>`, as in https://github.com/w3c/did-core/pull/785/files#r669609769

Reply:
Fixed in https://github.com/w3c/did-core/pull/785/commits/8c3b32f6b6876d28ba2e4a2463153383cb9f0b37.